aten/src/ATen/native/native_functions.yaml

```diff
@@ -5450,6 +5450,13 @@
     CPU: foreach_add_scalar_kernel_fallback
     CUDA: foreach_tensor_add_scalar_kernel_cuda
 
+- func: _foreach_add_.Scalar(Tensor[](a!) self, Scalar scalar) -> Tensor[](a!)
+  device_guard: False  
+  variants: function
+  dispatch:
+    CPU: foreach_add_scalar__kernel_fallback
```

Comment:
please put the underscore at the end of the name, i.e. `foreach_add_scalar_kernel_fallback_`, that's much easier to read.

Reply:
Done